proto/config.proto

```diff
@@ -195,6 +195,12 @@ message ProfileOptions
 	repeated GpioMappings gpioMappingsSets = 2 [(nanopb).max_count = 3];
 }
 
+message SplashFrame
+{
+	optional int32 delay = 1;
+	optional bytes frame = 2 [(nanopb).max_size = 1024];
```

Comment:
all frames together are 1024?

Reply:
No, each frame is 1036 bytes, these 1024 bytes  are the pixels for each one of them